src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,14 +1,31 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+import mate.academy.spring.validation.PasswordValidation;
+
+@PasswordValidation
 public class UserRequestDto {
+    @Email
     private String email;
+    @Size(min = 8, max = 16)
     private String password;
 
+    private String repeatPassword;
```

Comment:
```suggestion
    @Size(min = 8, max = 16)
    private String repeatPassword;
```
Would be good to add validation here too